README.md

```diff
@@ -42,9 +42,15 @@ First, open `./examples/standard/params.json` with your preferred text editor an
 
 Next, run the following:
 ```
-python src/main.py /examples/standard/params.json
+python src/main.py ./examples/standard/params.json
 ```
 
+Please note that this will invoke Chromer browser (headless chromedriver) that you have configured as part of prerequisites. When the browser comes up, please enter your Azure AD credentials, at present the workflow is divided into 2 distinct phases
```

Comment:
@bhavink-db Do we still need to have this section if we are are also approving the SP based method (that does not require headless chromedriver) in conjunction with this readme change?